about.html

```diff
@@ -0,0 +1,133 @@
+        <div class="guide_bar"></div>
+        <p>"There is only one boss. The Guest!"</p>
+        <div
+          class="card border mt-5 mb-5 p-5 line-height about-description bg-white rounded text-center"
+        >
+          <p>
+            Morning Due also known as Morning Due Guest Rooms, is a Kenyan
+            multinational hospitality chain of leased and franchised hotels,
+            homes and living spaces. Founded in 2022 by John Doe, initially
+            consisted mainly of budget hotels. The chain store serves both local
+            and international guest with hotels faclities at affordable rates.
+          </p>
+        </div>
+
+        <div class="text-center mx-5">
+          <p>
+            Please contact us for further inqueries about MorningDue.com
+            <br /><b>Email: morningdue@gmail.com</b>
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section id="about-middle">
+      <div class="about-middle text-center pt-5 pb-5 mt-5">
+        <h4>What Customers Say?</h4>
+        <div class="guide_bar"></div>
+        <p class="mx-3">
+          Take a look at what our recent customers are saying about their
+          experience at Morning Due.
+        </p>
+        <div class="container">
+          <div class="row">
+            <div class="col-md-6">
+              <div class="img-centered-text">
+                <p>
+                  <i
+                    >Nice place, really clean and pleasant rooms, but the thing
+                    that makes this such a good place to stay are the staff. All
+                    of the people are incredibly helpful and generous with their
+                    time and advice.<br />
+                    — Tove, stayed September 2022
+                  </i>
+                </p>
+              </div>
+            </div>
+
+            <div class="col-md-6">
+              <div class="img-centered-text">
+                <p>
+                  <i
+                    >We were blown away by our wonderful room with two poster
+                    beds, amazing garden views with the added bonus of Free
+                    WIFI. <br />
+                    — Jani, stayed September 2022
+                  </i>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="footer">
+      <div class="footer">
+        <h3 class="text-center display-4">Partner</h3>
+        <div class="guide_bar"></div>
+
+        <ul class="footer__container">
+          <li class="footer--item">Pixabay</li>
+          <li class="footer--item">Flaticon</li>
+          <li class="footer--item">Google</li>
+          <li class="footer--item">StackOverflow</li>
+          <li class="footer--item">StackExchange</li>
+        </ul>
+      </div>
+
+      <div class="rights text-center p-3">
+        <p class="mx-3">All Rights Reserved &copy;2022 Morning Due</p>
+      </div>
+    </section>
```

Comment:
- [ ] I think it would be better if you add a section in your **abiut.html** with a design like in the image below to make your page identical as much as possible to the expected design.

![Screenshot from 2022-09-14 20-11-01](https://user-images.githubusercontent.com/83042742/190230583-3ef5bb23-f620-4859-9970-cc9e641c23ff.png)

- [ ] Also make sure you use at least the correct **fonts(size && weight)** on each element text of your page. Check everything on the design and try to stick to it as much as possible